dsgrid/cli/registry.py

```diff
@@ -1753,16 +1753,24 @@ def remove_datasets(registry_manager: RegistryManager, dataset_ids: list[str]):
 @click.command(name="bulk-register", epilog=_bulk_register_epilog)
 @click.argument("registration_file", type=click.Path(exists=True))
 @click.option(
-    "-d",
-    "--base-data-dir",
+    "-D",
+    "--data-base-dir",
```

Comment:
I changed the bulk-register command to match the options used in the dataset-register command. It is a breaking change, but is better to be consistent.